python/lsst/ts/m2gui/controltab/tab_cell_status.py

```diff
@@ -33,7 +43,282 @@ class TabCellStatus(TabDefault):
         Table's title.
     model : `Model`
         Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
     """
 
     def __init__(self, title, model):
         super().__init__(title, model)
+
+        self._view_mirror = ViewMirror()
+        self._button_show_alias = set_button(
+            "Show Alias", self._callback_show_alias, is_checkable=True
+        )
+
+        self._forces = ActuatorForce()
+        self._figures = self._create_figures()
+
+        self._actuator_data_selection = self._create_actuator_data_selection()
+
+        # Internal layout
+        self.widget().setLayout(self._create_layout())
+
+        # Resize the dock to have a similar size of layout
+        self.resize(self.widget().sizeHint())
+
+        self._set_signal_detailed_force(
+            self.model.utility_monitor.signal_detailed_force
+        )
+
+    @Slot()
+    def _callback_show_alias(self):
+        """Callback of the show-alias button. The actuator will show the alias
+        or the actuator ID based on this."""
+
+        is_alias = self._button_show_alias.isChecked()
+        self._view_mirror.show_alias(is_alias)
+
+    def _create_figures(self):
+        """Create the figures to show the actuator forces.
+
+        Returns
+        -------
+        figures : `dict`
+            Figures of the actuator forces.
+        """
+
+        num_axial_actuator = NUM_ACTUATOR - NUM_TANGENT_LINK
+
+        label_x = "Actator ID"
```

Comment:
Act**u**ator ID

Reply:
Fixed in the commit: [af8c4ea](https://github.com/lsst-ts/ts_m2gui/pull/13/commits/af8c4eada9478d63f692c5cd50c53c057e3e2b34). Thanks!